src/pages/exercise2.jsx

```diff
@@ -0,0 +1,7 @@
+const exercise2 = () => {
```

Comment:
Change the naming of this component to "Exercise2" since react components should always be in PascalCase.